Gems/SceneProcessing/Code/Source/SceneBuilder/SceneBuilderWorker.cpp

```diff
@@ -43,6 +43,22 @@
 
 namespace SceneBuilder
 {
+    struct DebugScope final
+    {
+        DebugScope(bool isDebug)
```

Comment:
Does this only work in Debug builds? I wonder if it should be titled something to make it clear you can generate this info in non-Debug builds, too?

Reply:
it only works in the "debug output" mode, so thus the name "DebugScope" so maybe "DebugOutputScope" ?